tools/webdriver/webdriver/client.py

```diff
@@ -114,17 +114,17 @@ def perform(self):
         """Perform all queued actions."""
         self.session.actions.perform([self.dict])
 
-    def _key_action(self, subtype, value):
-        self._actions.append({"type": subtype, "value": value})
+    def _key_action(self, subtype, value, asyncDispatch=False):
```

Comment:
Looks like there are several lint errors about the name of this param (eg:
./webdriver/webdriver/client.py:120:49: N803 argument name 'asyncDispatch' should be lowercase)